test/test.js

```diff
@@ -628,6 +628,31 @@ describe('Page', function() {
     }));
   });
 
+  describe('Frame.setContent', function() {
```

Comment:
Are these tests exactly the same as the ones for the page.setContent?

In this case, I'd drop them to avoid repetition and save some time on test running.

Reply:
Yes. My thinking is that since both the Frame and Page APIs are publicly exposed, it made sense to test them both. However, I could merge the tests together to reduce the number of individual tests that run.

Let me know what you think :)

Reply:
> Yes. My thinking is that since both the Frame and Page APIs are publicly exposed, it made sense to test them both.

This is reasonable. We have a [simple coverage](https://github.com/GoogleChrome/puppeteer/blob/master/CONTRIBUTING.md#public-api-coverage) for the public API to enforce this.

> However, I could merge the tests together to reduce the number of individual tests that run.

Thanks, I appreciate it!